### PR TITLE
Cart 158 remove property

### PR DESCRIPTION
**Pull request by RomanFedytskyi**

### CODE REVIEW

File: dev/publicDev/properties/propertiesService.js [hunk 1]
```diff
@@ -30,8 +30,8 @@ function service ($http, TokenService) {
     getPhotoPath: function(photoPath) {
       return photoPath + '?' + TokenService.getTokenName() + '=' + TokenService.getToken();
     },
-    remove: function(id) {
-      return $http.delete('/api/property/' + id, {
+    delete: function(id) {
+      return $http.get('/api/property-delete/' + id, {
```
Comment (dmitrij-borchuk) [inline dev/publicDev/properties/propertiesService.js]: Why do you change "delete" method to "get"?

File: dev/routing/property.js [hunk 2]
```diff
@@ -137,7 +137,8 @@ module.exports = function (server) {
         }
       },
       handler: function (request, reply) {
-        DAL.properties.remove(request.params.id, function (err, docs) {
+        console.log(request.params, '-----------------')
```
Comment (dmitrij-borchuk) [inline dev/routing/property.js]: Remove console.log

**Review by dmitrij-borchuk — CHANGES_REQUESTED** [review]

File: dev/publicDev/properties/propertiesService.js (hunk 1 above)
Comment (RomanFedytskyi) [reply dev/publicDev/properties/propertiesService.js]: fixed

File: dev/routing/property.js (hunk 2 above)
Comment (RomanFedytskyi) [reply dev/routing/property.js]: fixed

**Pull request merged**